src/js/view/controls/components/menu/category-button.js

```diff
@@ -0,0 +1,39 @@
+import button from 'view/controls/components/button';
+import { cloneIcon } from 'view/controls/icons';
+import { SimpleTooltip } from 'view/controls/components/simple-tooltip';
+import { createElement } from 'utils/dom';
+
+const categoryButton = (menu, localizedName) => {
+    const name = menu.name;
+    const icons = {
+        captions: 'cc-off',
+        audioTracks: 'audio-tracks',
+        quality: 'quality-100',
+        playbackRates: 'playback-rate',
+    };
+
+    // Allows override from Commercial Sharing menu
+    let icon = icons[name];
+    if (!icon) {
+        icon = icons.quality;
```

Comment:
Is there a reason the quality icon is the default one used if one is not provided? I'm just curious.

Reply:
Made it easier to test and develop, ensures there's always an icon (it's the only UI for category buttons visible without interacting directly with the button). That being said, after reading your and Joe's comments, I've decided to make it return when there isn't an icon instead.